lib/ClangImporter/ImportDecl.cpp

```diff
@@ -2036,11 +2041,76 @@ namespace {
         enumDecl->addMember(rawValueGetter);
         enumDecl->addMember(rawValue);
         enumDecl->addMember(rawValueBinding);
-        
         result = enumDecl;
+
+        // Set up the domain error member to be lazily added
+        if (enumInfo.isErrorEnum()) {
+          auto clangDomainIdent = enumInfo.getErrorDomain();
```

Comment:
That would be ideal, but the attribute tablegen backend currently does not allow the storing of arbitrary decls in attributes. I can certainly add that functionality, but probably after I get this essential functionality fixed. Once this gets through, I'll open a radar for that.